### PR TITLE
[Heartbeat] Fix zip monitors 

### PR DESCRIPTION
**Pull request by andrewvc**
Fixes them by changing the check group after summar…y docs

Fixes https://github.com/elastic/beats/issues/33698

No tests, etc. because this feature will be removed in 8.7

Tested with:

```yaml
- name: TDZIP
  id: tdzip
  type: browser
  schedule: "@every 3m"
  tags: todos-app
  params:
    url: "https://elastic.github.io/synthetics-demo/"
  source:
    zip_url: 
      url: "https://github.com/elastic/synthetics-demo/archive/refs/heads/main.zip"
      folder: "todos"
```

Run with, from `x-pack/heartbeat` folder:

` mage build && ELASTIC_SYNTHETICS_CAPABLE=true ./heartbeat -e -E cloud.id=$CLOUD_ID -E cloud.auth=$CLOUD_AUTH -E output.elasticsearch.allow_older_versions=true`

### CODE REVIEW

File: x-pack/heartbeat/monitors.d/zipurl.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+- name: TDZIP
```
Comment (emilioalvap) [inline x-pack/heartbeat/monitors.d/zipurl.yml:2]: Should this file be commited?

Comment (andrewvc) [reply x-pack/heartbeat/monitors.d/zipurl.yml:2]: ack, nope

File: x-pack/heartbeat/monitors/browser/synthexec/enrich.go [hunk 2]
```diff
@@ -84,6 +92,7 @@ func (je *journeyEnricher) enrich(event *beat.Event, se *SynthEvent) error {
 		// Record start and end so we can calculate journey duration accurately later
 		switch se.Type {
 		case JourneyStart:
+			je.streamEnricher.checkGroup = makeUuid()
```
Comment (emilioalvap) [inline x-pack/heartbeat/monitors/browser/synthexec/enrich.go:95]:
```suggestion
			// TODO: Remove this when zip monitors are removed and we have 1:1 monitor / journey
			je.streamEnricher.checkGroup = makeUuid()
```

**Review by emilioalvap — APPROVED** [review]
LGTM

**Pull request merged**